### DISCUSSION TITLE
[Calendar component ]: Change non-working days

### DISCUSSION THREAD

**Discussion by DamyanKolev** [UI5/webcomponents-react#5914] [Q&A]
My question is, is there a way to change the non-working days in the Calendar component. I want to add  holidays as non-working days in the component.

**Comment by Lukas742** [comment]
Hi @DamyanKolev 

usually you should be able to achieve this by using the `"NonWorking"` `type` of the `SpecialCalendarDate` component, but it turns out that it currently doesn't apply any style changes for some of the types. I've already created an [issue](https://github.com/SAP/ui5-webcomponents/issues/9216) in the ui5-webcomponents repo, so unfortunately you'll have to wait until this is fixed there. If you have additional info, please feel free to add it to the issue.

Once the bug has been fixed, you can use the `SpecialCalendarDate` and the `type` attribute like this: https://stackblitz.com/edit/github-lbjekt?file=src%2FApp.tsx
